kardia/client.go

```diff
@@ -50,6 +50,9 @@ type ClientInterface interface {
 
 	// params related methods
 	GetMaxProposers(ctx context.Context) (int64, error)
+	GetParams(ctx context.Context) (map[string]string, error)
+	GetProposalDetails(ctx context.Context, proposalId *big.Int) (*types.ProposalDetail, error)
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>interface method parameter `proposalId` should be `proposalID` (golint)